### PR TITLE
Alpha version - Update UG edit, save, help

### LINKED ISSUE #199: Alpha version - Update UG edit, save, help
Update `edit` and `help` commands and `save` information in UG to match our implementation

### PR DESCRIPTION
**Pull request by sannie-beep**
fixes #199

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ _Refer to the [Features](#features) section for details of each command!_
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a popup window with a link to the User Guide.
```
Comment (PrishaVP) [inline docs/UserGuide.md:97]: maybe add "clickable" in front of link just to make it very clear how it's been improved

File: docs/UserGuide.md [hunk 2]
```diff
@@ -126,18 +126,24 @@ Format: `list`
...
+Format: `edit INDEX [n/NAME] [e/EMAIL] [g/GENDER] [a/AGE] [t/ADD-TAG] [-t/DELETE-TAG]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+* When editing tags:
+  * Prefix:`t/ADD-TAG]`
```
Comment (PrishaVP) [inline docs/UserGuide.md:134]: was having only 1 half of the square bracket ] intentional? if so, why?

Comment (PrishaVP) [reply docs/UserGuide.md:134]: if it was unintentional, the delete tag prefix u wrote below also has only ] if u want to change that

File: docs/UserGuide.md [hunk 3]
```diff
@@ -126,18 +126,24 @@ Format: `list`
 
 Edits an existing person in the address book.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [e/EMAIL] [g/GENDER] [a/AGE] [t/ADD-TAG] [-t/DELETE-TAG]…​`
 
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```
Comment (PrishaVP) [inline docs/UserGuide.md:131]: can add this to a new bullet: The index **must be a positive integer** 1, 2, 3, …​
(following what other commands' have)

File: docs/UserGuide.md (hunk 2 above)
Comment (sannie-beep) [reply docs/UserGuide.md:134]: Oops unintentional. Will fix, thanks!

**Comment by PrishaVP** [comment]
LGTM

**Pull request merged**